omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptReferenceTermResource1_9.java

```diff
@@ -163,9 +164,14 @@ protected PageableResult doSearch(RequestContext context) {
 		String query = context.getParameter("q");
 		if (query == null)
 			return new EmptySearchResult();
-		List<ConceptReferenceTerm> terms = cs.getConceptReferenceTerms(query, null, context.getStartIndex(),
+        String sourceUuid = context.getParameter("source");

```

Comment:
@dkayiwa I am debugging RestServiceImpl#getSearchHandlerMethod, I've sent search request with searchParameters `q` and `source`, and at line [495](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImpl.java#L496) it starts iterating over them. In first iteration it takes `q` parameter, and returns null at line [499](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImpl.java#L499), because in `searchHandlersByParameter` map for `/conceptreferenceterm` the `q` parameter is not declared. 
When I send request like `http://localhost:8080/openmrs/ws/rest/v1/conceptreferenceterm?source=2b3c09b4-768a-102f-83f4-12313b04a615` RestService finds SearchHandler, but of course it returns empty results, as there is no query.
Same situation happens for other resources.